data/json/items/magazine/weldgas.json

```diff
@@ -0,0 +1,34 @@
+[
+  {
+    "id": "tinyweldtank",
+    "type": "MAGAZINE",
+    "name": "small welding tank",
+    "description": "A compact 5-litre steel cylinder designed for holding pressurised welding gasses.  It is marked with several faded and now unreadable symbols.",
+    "material": "steel",
+    "symbol": "#",
+    "color": "brown",
+    "weight": 1500,
+    "volume": 4,
+    "price": 5600,
+    "ammo_type": "weldgas",
+    "capacity": 60,
+    "count": 60,
+    "flags": [ "NO_UNLOAD", "NO_RELOAD" ]
+  },
+  {
+    "id": "weldtank",
+    "type": "MAGAZINE",
+    "name": "welding tank",
+    "description": "A large 20-litre steel cylinder designed for holding pressurised welding gasses.  It is marked with several faded and now unreadable symbols.",
```

Comment:
Realistically, you wouldn't ever have a pressurized container of oxygen-acetylene mix, because of what a huge explosion hazard it is. And also because you control the temperature of the flame by adjusting the oxygen-acetylene ratio.
As a placeholder for actually using multiple types of ammo, I'd suggest changing the magazine description. Make it a pair of tanks instead.